test/test-data-unicode-set.json

```diff
@@ -914,76 +914,10 @@
     "raw": "[[a-z]&&[p-s]]"
   },
   "[AB&&CD]": {
```

Comment:
This is invalid in unicode sets mode. V8 12.6 also throws "Invalid set operation in character class".